quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
Yes, all hosts.

For a host in a domain it would only write something like:
```pan
"/metadata/template/branch/name" = "prod";
"/metadata/template/branch/type" = "domain";
```

Implementing the git commit id would probably be best supported in the compiler IMHO.